tests/conftest.py

```diff
@@ -56,6 +57,14 @@ def setup_db_and_server(
     metadata.drop_all(engine)
 
 
+@pytest.fixture(autouse=True)
```

Comment:
what does `autouse=True` do? do we want to default to not testing auth? we could instead maybe have a logged in user as fixture instead